examples/runner-certificates/README.md

```diff
@@ -0,0 +1,216 @@
+# Example - Spot Runner - Private subnet
```

Comment:
praise: appreciated! Having an example makes life easier. Thanks for that.

Reply:
@kayman-mk thanks for the great feedback :) 

I agree and I've updated for all suggestions, please check again.